pkg/services/playlist/model.go

```diff
@@ -82,9 +94,27 @@ type GetPlaylistItemsByUidQuery struct {
 	OrgId       int64
 }
 
-func PlaylistToResource(p PlaylistDTO) playlist.K8sResource {
-	copy := p
-	r := playlist.NewK8sResource(p.Uid, &copy)
-	copy.Uid = "" // remove it from the payload
-	return r
+func ConvertToK8sResource(v *Playlist, items []PlaylistItemDTO) *playlist.Playlist {
+	return &playlist.Playlist{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Playlist",
+			APIVersion: playlist.APIVersion,
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              v.UID,
+			UID:               types.UID(v.UID),
+			ResourceVersion:   fmt.Sprintf("%d", v.UpdatedAt),
+			CreationTimestamp: metav1.NewTime(time.UnixMilli(v.CreatedAt)),
+			Namespace:         fmt.Sprintf("org-%d", v.OrgId),
+			// Annotations: map[string]string{
```

Comment:
?

Reply:
Can you either remove the commented-out code, or add some comments / TODOs as explanations?